CHANGELOG.adoc

```diff
@@ -95,6 +95,7 @@ The new endpoints are:
 ** deleteBranch (`DELETE /api/rest/projects/{projectId}/branches/{branchId}`): Delete the branch with the given Id in the given project. There is only one branch per project in Sirius Web for now, so the default implementation of this method does nothing.
 - https://github.com/eclipse-sirius/sirius-web/issues/4440[#4440] [sirius-web] Add the representation description name in details view
 - https://github.com/eclipse-sirius/sirius-web/issues/4425[#4425] [table] Add new table cell widget description for textarea
+- https://github.com/eclipse-sirius/sirius-web/issues/4462[#4462] [sirius-web] Add support for library publication and visualization.
```

Comment:
I don't know if I should list some key changes related to this work (e.g. new database table, new `/libraries` page etc)

Reply:
It would be nice, the goal of the CHANGELOG is not to be a list of Github issue titles but instead a mean of communication to other members of our community. In the case of a contribution like this one some additional details would be nice.